java/all/src/main/scala/org/typelevel/otel4s/java/OtelJava.scala

```diff
@@ -46,10 +46,12 @@ object OtelJava {
     * @return
     *   An effect of an [[org.typelevel.otel4s.Otel4s]] resource.
     */
-  def forSync[F[_]: LiftIO: Sync](jOtel: JOpenTelemetry): F[Otel4s[F]] = {
+  def forSync[F[_]: Sync: Local[*[_], TraceScope.Scope]](
```

Comment:
This is where Scope starts to leak.  I think we can fix this with some Greek letters.

[edit] Well, probably not: if `F` is Kleisli, we start to publicly return a bunch of functions of Scope.